Require/index.js

```diff
@@ -211,4 +223,28 @@ describe("TNS require", function () {
         require('./RequireJsonCorruptFile');
         expect(TNSGetOutput()).toMatch(/JSON Parse error: Unterminated string$|Unexpected token/)
     });
+
+    it("shouldn't load invalid JSON file", function () {
+		require("./RequireJsonCorruptFile1");
+		expect(TNSGetOutput()).toMatch(/JSON Parse error: Unterminated string$|Unexpected token/)
+	});
+    
+    it("when using global in a module global should be defined", function () {
+		var module = require("./GlobalIsDefined/module");
```

Comment:
There is also no harm in checking three times, but since checking several times doesn't bring any value to your test I don't see a reason to do it. 
Do it as you wish though, I am just expressing my opinion :)